src/molecules/header/Header.module.scss

```diff
@@ -1,27 +1,51 @@
-@import 'settings/variables.scss';
-@import 'styles/colors.module.scss';
+@import '../../settings/variables.scss';
+@import '../../styles/colors.module.scss';
```

Comment:
Why `absolute` paths don't work in SCSS?

Reply:
because it needs webpack.config.js, but there is alternative using ~settings/variables.scss this also works, so what do you think? add webpack.config.js oder using ~ ?

Reply:
would webpack.config.js have to override everything that React currently have under the hood?

Reply:
I try again to remove ../../ without adding anything. it works now